deploy/config/k8s/test-env.sh.yml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       serviceAccountName: ${APP_NAMESPACE_NAME}-sa
       containers:
-      - image: debian
+      - image: '${PULL_DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}/debian:latest'
```

Comment:
I think we also need this in `secrets-provider-init-container.sh.yml`

Reply:
I thought so but turns out that manifest is entirely for development purposes so we can't make assumptions about the image registry. You'll see that `secrets-provider-init-container.sh.yml` is pulling in `debian` directly as well as `secrets-provider-for-k8s:latest`, see https://github.com/cyberark/secrets-provider-for-k8s/blob/main/deploy/dev/config/k8s/secrets-provider-init-container.sh.yml#L50. So we can leave that one out.